src/unix/stream.c

```diff
@@ -301,7 +301,9 @@ int uv__stream_try_select(uv_stream_t* stream, int* fd) {
   if (fds[1] > max_fd)
     max_fd = fds[1];
 
-  sread_sz = (max_fd + NBBY) / NBBY;
+  sread_sz = (max_fd + 1) / NBBY;
+  if (sread_sz % sizeof(uint32_t) != 0)
+    sread_sz += sizeof(uint32_t) - (sread_sz % sizeof(uint32_t));
```

Comment:
@bnoordhuis :+1: (I was tempted to introduce a macro like that, but went with minimal changes as I'm not yet familiar with the libuv code-base.)